### ISSUE TITLE
[Feature Request] Python integration? :)

### ISSUE THREAD

**Issue by ItsCubeTime** [makspll/bevy_mod_scripting#131]

**Comment by ItsCubeTime** [comment]
I'm glad! Please let me know if anyone picks this up. Theres a great lack of high performance game engines with some form of runtime Python API.

**Comment by mad-moo** [comment]
Have you considered using PyPy instead of CPython? It's not 100% compatible, but often close enough, and much faster. Sounds like a great fit for a game engine

**Comment by ItsCubeTime** [comment]
@mad-moo For me personally, any compatibility issues with CPython could prove problematic as I would want to be able to make good use of the full ecosystem on https://www.pypi.org/ 

But since all thats needed is really just some kind of API and since Python itself doesn't require any build step, maybe it would be possible to support several Python runtimes?

**Comment by mad-moo** [comment]
You're right. The ecosystem is of course one of the main points of Python. But in the context of a Game, speed is too. I'd say PyPy is worth considering.

> But since all thats needed is really just some kind of API and since Python itself doesn't require any build step, maybe it would be possible to support several Python runtimes?

That's  a good point. Given that PyPy is _trying_ to be compatible, it may be possible to support both without much code - I don't know
